posthog/temporal/batch_exports/postgres_batch_export.py

```diff
@@ -675,7 +679,14 @@ async def insert_into_postgres_activity(inputs: PostgresInsertInputs) -> Records
             try:
                 columns = await pg_client.aget_table_columns(inputs.schema, inputs.table_name)
                 table_fields = [field for field in table_fields if field[0] in columns]
+            except psycopg.errors.InsufficientPrivilege:
+                await internal_logger.awarning(
+                    "Insufficient privileges to get table columns for table '%s.%s'; will assume all columns are present",
+                    schema=inputs.schema,
+                    table_name=inputs.table_name,
+                )
```

Comment:
We could probably make this a public log if we indicate users how to resolve this:
* Grant us permissions.
* Ensure table is updated to latest schema.

Reply:
yeah that's a good idea, thanks

Reply:
have updated